src/main/scala/cromwell/util/Encryption.scala

```diff
@@ -23,6 +23,22 @@ case object Aes256Cbc {
 
   val ranGen = new SecureRandom()
 
+  lazy val isAvailable = {
+    try {
+      // TODO: Easier way to test if using this key size is supported?
+      val key = new Array[Byte](keySize)
+      val iv = new Array[Byte](blockSize / 8)
+      ranGen.nextBytes(key)
+      ranGen.nextBytes(iv)
+      init(Cipher.ENCRYPT_MODE, key, iv)
+      init(Cipher.DECRYPT_MODE, key, iv)
+      true
+    } catch {
+      case e: InvalidKeyException =>
+        false
+    }
+  }
```

Comment:
Should we support a fail fast for missing AES 256 in production code?  That could basically just call this method and bomb out if it returns false.